eclass/user-info.eclass

```diff
@@ -37,18 +37,27 @@ egetent() {
 		# lookup by uid/gid
 		local opts
 		if [[ ${key} == [[:digit:]]* ]] ; then
-			[[ ${db} == "user" ]] && opts="-u" || opts="-g"
+			[[ ${db} == "user" ]] && opts=( "-u" ) || opts=( "-g" )
 		fi
 
+		# Handle different ROOT
+		[[ $ROOT == "/" ]] || opts+=( -R "${ROOT}" )
+
 		pw show ${db} ${opts} "${key}" -q
 		;;
 	*-openbsd*)
-		grep "${key}:\*:" /etc/${db}
+		grep "${key}:\*:" ${ROOT}/etc/${db}
```

Comment:
```suggestion
		grep "${key}:\*:" "${ROOT}"/etc/${db}
```

should this be quoted?

Reply:
Good catch, I will quote the whole string